tutorials/tts/FastPitch_E2E_Training.ipynb

```diff
@@ -0,0 +1,483 @@
+    "that you want to use the \"Run All Cells\" (or similar) option.\n",
+    "\"\"\"\n",
+    "# exit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ZivXzmq0YYLj"
+   },
+   "source": [
+    "# FastPitch End-to-end and NeMo\n",
+    "\n",
+    "FastPitch End-to-end is a joint model which combines FastPitch and HiFi-GAN to converts text characters directly into an audio sample. \n",
+    "\n",
+    "FastPitch End-to-end like most NeMo models are defined as a LightningModule, allowing for easy training via PyTorch Lightning, and parameterized by a configuration, currently defined via a yaml file and loading using Hydra."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "HEvdSU5WYZbj"
+   },
+   "outputs": [],
+   "source": [
+    "# Load the FastPitch End-to-end model\n",
+    "from nemo.collections.tts.models import FastPitchE2EModel\n",
+    "from nemo.collections.tts.models.base import TextToWaveform\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "TextToWaveform in NeMo have two helper functions:\n",
+    "   1. ```python\n",
+    "      parse(self, str_input: str),\n",
+    "      ``` \n",
+    "      which takes an English string and produces a token tensor\n",
+    "\n",
+    "\n",
+    "\n",
+    "   2. ```python\n",
+    "      convert_text_to_waveform(self, *, tokens),\n",
+    "      ```\n",
+    "      which takes the token tensor and generates an audio sample\n",
+    "Let's try it out"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zZ90eCfdrNIf"
+   },
+   "source": [
+    "# Training\n",
+    "\n",
+    "Let's see how to train a FastPitch End-to-end Model\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "7rHG-LERrPRY"
+   },
+   "outputs": [],
+   "source": [
+    "# NeMo's training scripts are stored inside the examples/ folder. Let's grab the fastpitch_e2e.py file\n",
+    "# as well as the fastpitch_e2e.yaml file\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/tts/fastpitch_e2e.py\n",
+    "!(mkdir -p conf \\\n",
+    "  && cd conf \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/tts/conf/fastpitch_e2e.yaml \\\n",
+    "  && cd ..)\n",
+    "\n",
+    "# additional files\n",
+    "!mkdir -p tts_dataset_files && cd tts_dataset_files \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/ipa_cmudict-0.7b_nv23.01.txt \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/heteronyms-052722 \\\n",
+    "&& cd ..\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Upv_LxBIsC51"
+   },
+   "source": [
+    "# Runner\n",
+    "\n",
+    "Let's take a look at the fastpitch_e2e.py file\n",
+    "\n",
```

Comment:
The following sections are very similar to the VITS tutorial, can you deduplicate the explanations? Also need to remove some references to VITS.